src/main/java/frc/robot/subsystems/gpm/Intake.java

```diff
@@ -133,11 +119,9 @@ private void publish() {
     public void setMode(Mode mode) {
         this.mode = mode;
 
-        // set the motor powers to be the value appropriate for this mode
```

Comment:
Why are you deleting comments that describe what the code does? This one might be unnecessary because the line it describes is self-explanatory, but you still usually shouldn't delete comments.